engine/src/main/java/com/sandboni/core/engine/sta/operation/ReflectionCallTestsOperation.java

```diff
@@ -0,0 +1,30 @@
+package com.sandboni.core.engine.sta.operation;
+
+import com.sandboni.core.engine.sta.graph.Edge;
+import com.sandboni.core.engine.sta.graph.LinkType;
+import com.sandboni.core.engine.sta.graph.vertex.TestVertex;
+import com.sandboni.core.engine.sta.graph.vertex.Vertex;
+import org.jgrapht.Graph;
+
+import java.util.Set;
+
+import static com.sandboni.core.engine.common.StreamHelper.emptyIfFalse;
+import static com.sandboni.core.engine.sta.graph.vertex.VertexInitTypes.REFLECTION_CALL_VERTEX;
+
+public class ReflectionCallTestsOperation extends AbstractGraphOperation<SetResult<TestVertex>> {
+
+    private final Set<TestVertex> allTests;
+
+    public ReflectionCallTestsOperation(Set<TestVertex> allTests) {
+        this.allTests = allTests;
```

Comment:
Create a copy allTests Set, this avoid to have "scaped" parameters that could lead to hard to find bugs:
 this.allTests = new HashSet<>(allTests);

Reply:
done